pkg/rpc/server/server.go

```diff
@@ -2022,7 +2019,7 @@ func (s *Server) subscribe(reqParams request.Params, sub *subscriber) (interface
 		}
 	}
 	if id == len(sub.feeds) {
-		return nil, response.NewInternalServerError("maximum number of subscriptions is reached", fmt.Errorf("%d subscriptions are allowed at max", len(sub.feeds)))
+		return nil, response.NewInternalServerError(fmt.Errorf("maximum number of subscriptions is reached: %d", len(sub.feeds)))
```

Comment:
Do we need to disclose this data to the user?

Reply:
I thought it might be useful, but OK, fixed.

Reply:
It's leaking configuration data to the client, which can be a security issue.